setup.py

```diff
@@ -21,17 +15,10 @@
     description="Runs a thing that analyzes experiments",
     url="https://github.com/mozilla/pensieve",
     packages=["pensieve", "pensieve.tests"],
-    install_requires=[
-        "attrs",
-        "cattrs",
-        "incremental",
-        "pytz",
-        "requests",
-    ],
+    install_requires=text_from_file("requirements.txt").strip().split("\n"),
     setup_requires=["incremental"],
-    tests_require=test_dependencies,
-    extras_require=extras,
```

Comment:
iirc removing the extras_require will break the tox.ini, which relies on installing the `testing` extra. (In contrast, the `tests_require` key is kind of useless; we could avoid specifying it.)